docs/source/testing/trip_test.rst

```diff
@@ -0,0 +1,99 @@
+Trip Tests
+==============
+
+Summary
+-------
+The trip test program takes two commit IDs as input, and compares the results of Isca's test cases using the two versions of Isca defined by the commit IDs. A comparison is then made between the output from each of the test cases. The output of the program shows which test cases produced bit-wise identical results and which did not.
+
+
+Why do we ask you to run the trip tests?
+-----------------------------------------
+
+We are very keen for people to contribute their modifications and enhancements back into Isca's main version. However, before we accept your modifications, we require you to run the trip tests. By running these tests, we can check whether your modifications changes the results of any of the test cases.
+
+We want to check this for three principal reasons:
+
+    1. If you add a new feature, we want to make sure it is turned off by default. This is so that an existing Isca user who doesn't know about the feature isn't forced to use it when they upgrade to the newest Isca version. This prevents confusion where two versions of Isca give different answers for the same experimental configuration.
+    2. If your modification contains a bug fix, we want to know which test cases are affected. We also want to know which P/R resulted in the bug fix.
+    3. You may think your modifications won't have changed Isca's results, but we want to make sure that's the case. The trip tests are very good at highlighting unexpected consequences of your changes, and help us keep to a high-standard of reproducibility.
+
+
+How do I run the trip tests?
+---------------------------------
+
```

Comment:
Before this you need the commit id. Suggest you say directly that you need to locate them.

Reply:
Also need to say somewhere that we need all the trip tests to pass, unless you can satisfactorily explain why they fail (ie a big fix). When you show the -e options (without saying this first) it makes it look like you can pick and choose which tests to run.